### ISSUE TITLE
Zero Health bug (rare!) BS-221

### ISSUE THREAD

**Issue by Niklas-Eckhoff** [punyhumangames/BladeSymphony#105]
While dueling on an FFA server, my opponent did not die even though my attack reduced the character's health to zero. My opponent was unable to attack. His attacks looked like Katana Feints, but there was no sparkling.

Note that while this is a rare bug, this is not the first time I've seen this happening either.

**Unknown method of reproduction.**

### Video proof of the bug (click the picture):
[![Join duel in grabbed state](https://img.youtube.com/vi/0IY8cZEic-A/0.jpg)](https://youtu.be/0IY8cZEic-A)

**Comment by lkirbyPH** [comment]
Hey there! We are now tracking this bug. Our internal code for this issue is BS-221

**Comment by lkirbyPH** [comment]
@Niky1 have you experienced this bug lately?

**Comment by Niklas-Eckhoff** [comment]
@lkirbyPH No, I haven't, but it could still be in the game. On the other hand it's so rare and not gamebreaking, just a little weird, so it's probably not worth fixing (and I wouldn't know how to reproduce it anyway, so there's that). It should be ok to close this thread. I'll tell you if I find a way to reproduce it or any helpful information.

**Comment by lkirbyPH** [comment]
Awesome. I really appreciate that!
